app/components/root/TeacherListSearch.tsx

```
@@ -50,8 +61,9 @@ function TeacherListSearch({
                   closeModal();
                 },
                 onError: (error) => {
-                  const errorMessage = "알림톡 발송 중 문제가 발생했습니다. 잠시 후 다시 시도해주세요.";
-                  console.error('매칭 알림톡 발송 실패:', error);
+                  const errorMessage =
+                    "알림톡 발송 중 문제가 발생했습니다. 잠시 후 다시 시도해주세요.";
+                  console.error("매칭 알림톡 발송 실패:", error);
```

ESLint

[error] Unexpected console statement.

(no-console)